src/components/layout/Navbar/NavGroup/NavGroup.tsx

```diff
@@ -9,6 +10,8 @@ const NavGroup = () => {
   const isMobile = useIsMobile();
   const { wallet, walletAddress } = useWalletAddress();
 
+  const [{}, dispatchGlobalState] = useGlobalState(); // eslint-disable-line
```

Comment:
I actually like it because it looks a bit cleaner, my only argument is I think the eslint this is more readable, and someone might not understand why theres an underscore there - I actually dont understand why that works myself.

Reply:
as long as we're consistent - indifferent on either pattern!